app/models/comment.rb

```diff
@@ -64,6 +75,12 @@ def initialize(person, target, text)
       super(person, target)
     end
 
+    def build(*args)
+      super.tap do |comment|
+        comment.filter_mentions(comment.people_allowed_to_be_mentioned)
```

Comment:
Yes, maybe status messages are also not properly filtered on receive. It was just an idea, that we may should also filter on receive, so we don't send notifications, if the user can't see the post. 😃